scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/InProcessTaskExecutor.java

```diff
@@ -191,6 +197,37 @@ public TaskResultImpl execute(TaskContext taskContext, PrintStream output, Print
         }
     }
 
+    private void saveScriptAsFile(String path, Script<?> script, TaskContext taskContext) throws Throwable {
```

Comment:
If there are other exception than TaskException I think that they should be handle in the private method otherwise I think it better to throw a TaskException than a throwable.